==> Consul Connect proxy starting...
    Configuration mode: Agent API
        Sidecar for ID: counting-1
              Proxy ID: counting-1-sidecar-proxy

==> Log data will now stream in as it occurs:

    2024-05-31T15:31:42.544+0800 [INFO]  proxy: Proxy loaded config and ready to serve
    2024-05-31T15:31:42.544+0800 [INFO]  proxy: Parsed TLS identity: uri=spiffe://06f214b1-5b8e-a5fa-797a-1d0cdb8d29b8.consul/ns/default/dc/ua-test/svc/counting
    2024-05-31T15:31:42.544+0800 [INFO]  proxy: Starting listener: listener="public listener" bind_addr=0.0.0.0:21000
    2024-05-31T15:31:51.265+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:32:01.266+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:32:11.267+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:32:21.269+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:32:31.270+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:32:41.271+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:32:51.272+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:33:01.273+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:33:11.274+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:33:21.275+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:33:31.276+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:33:41.276+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:33:51.277+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:34:01.279+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:34:11.279+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:34:21.281+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:34:31.281+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:34:41.282+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:34:51.283+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:35:01.285+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:35:11.286+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:35:21.286+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:35:31.287+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:35:41.288+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:35:51.288+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:36:01.290+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:36:11.293+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:36:21.293+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:36:31.295+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:36:41.296+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:36:51.296+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:37:01.297+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:37:11.299+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:37:21.300+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:37:31.302+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:37:41.303+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:37:51.305+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:38:01.306+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:38:11.308+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:38:21.308+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:38:31.310+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:38:41.311+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:38:51.313+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:39:01.314+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:39:11.314+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:39:21.315+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:39:31.316+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:39:41.317+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:39:51.318+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:40:01.320+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:40:15.600+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:40:25.601+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:40:35.603+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:40:45.603+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:40:55.605+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:41:05.608+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:41:15.609+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:41:25.609+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:41:35.609+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:41:45.610+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:41:55.611+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:42:05.613+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:42:15.615+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:42:25.617+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:42:35.618+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:42:45.618+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:42:55.619+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:43:05.621+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:43:15.621+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:43:25.623+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:43:35.625+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:43:45.625+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:43:56.247+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:44:06.249+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:44:16.249+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:44:26.251+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:44:36.253+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:44:46.254+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:44:56.254+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:45:06.256+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:45:16.258+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:45:26.259+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:45:36.260+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:45:46.261+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:45:56.263+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:46:06.264+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:46:16.266+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:46:26.267+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:46:36.268+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:46:46.270+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:46:56.272+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:47:06.273+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:47:16.273+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:47:26.275+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:47:36.275+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:47:46.276+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:47:56.277+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:48:06.278+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:48:16.279+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:48:26.281+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:48:36.281+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:48:46.283+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:48:56.284+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:49:06.286+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:49:16.287+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:49:26.288+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:49:36.289+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:49:46.291+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:49:56.292+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:50:06.293+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:50:16.294+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:50:26.295+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:50:36.296+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:50:46.296+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:50:56.298+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:51:06.299+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:51:16.300+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:51:26.301+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:51:36.302+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:51:46.304+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:51:56.305+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:52:06.306+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:52:16.307+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:52:26.309+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:52:36.311+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:52:46.312+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:52:56.314+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:53:06.316+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:53:16.318+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:53:26.320+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:53:36.323+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:53:46.325+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:53:56.327+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:54:06.327+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:54:16.328+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:54:26.329+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:54:36.331+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:54:46.332+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:54:56.333+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:55:06.334+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:55:16.336+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:55:26.339+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:55:36.341+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:55:46.341+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:55:56.343+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:56:06.343+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:56:16.345+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:56:26.346+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:56:36.348+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:56:46.350+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:56:56.352+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:57:06.353+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:57:16.355+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:57:26.357+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:57:36.358+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:57:46.359+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:57:56.361+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:58:06.362+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:58:16.363+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:58:26.364+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:58:36.366+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:58:46.367+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:58:56.369+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:59:06.369+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:59:16.371+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:59:26.371+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:59:36.373+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:59:46.375+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T15:59:56.377+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:00:06.378+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:00:16.380+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:00:26.380+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:00:36.381+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:00:46.382+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:00:56.384+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:01:02.297+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:01:12.299+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:01:22.303+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:01:32.305+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:01:42.306+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:01:52.307+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:02:02.308+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:02:12.310+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:02:22.312+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:02:32.313+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:02:42.314+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:02:52.315+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:03:02.316+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:03:12.317+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:03:22.319+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:03:32.320+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:03:42.322+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:03:52.325+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:04:02.326+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:04:12.327+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:04:22.328+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:04:32.330+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:04:42.332+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:04:52.333+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:05:02.334+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:05:12.335+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:05:22.336+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:05:32.338+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:05:42.339+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:05:52.341+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:06:02.342+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:06:12.343+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:06:22.345+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:06:32.347+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:06:42.347+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
    2024-05-31T16:06:52.348+0800 [ERROR] proxy.inbound: failed to dial: error="dial tcp 127.0.0.1:9003: connect: connection refused"
